content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -312,7 +312,7 @@ END CATCH
 <ButtonLink
   color="dark"
   type="primary"
-  href="./connect-your-database-node-postgres"
+  href="https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-node-postgres"
   arrowLeft
 >
   Connect your database
```

Comment:
Related to @janpio, I checked the page, here for example https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgres

Then let's find the button where the link is changed by the PR at the bottom and it's `href` value is
`/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-postgres/`
Which is indeed relative and could be made absolute by our tooling.
I also see the non-canonicalized problem where the href value on current website has an extra ending `/` at the end and the canonical doesn't.

Canonical value:
`[href=""](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgres)`

So 2 different problems where both would probably need some tooling investigation/fix from us?

Reply:
The relative link is no problem, and the trailing slash is easy to remove if that is what the problem is about. We just need to update our `ButtonLink` component probably.

Reply:
Based on the problem description, I opened issue https://github.com/prisma/docs/issues/2500 so this can properly be fixed. All the changes to the `ButtonLink` component usage would then be not necessary.